### PR TITLE
Removed container url and use resolved url from container in relative loader

### PR DESCRIPTION
**Pull request by jatgarg**
1.) Remove container url from container as we don't need it anymore to route request to base container.

### CODE REVIEW

File: packages/loader/container-loader/src/loader.ts [hunk 1]
```diff
@@ -70,34 +62,25 @@ export class RelativeLoader extends EventEmitter implements ILoader {
...
         if (request.url.startsWith("/")) {
-            // If no headers are set that require a reload make use of the same object
-            const container = await this.containerDeferred.promise;
-            return container;
+            if (canUseCache(request)) {
+                return this.container;
+            } else {
+                const resolvedUrl = this.container.resolvedUrl;
+                ensureFluidResolvedUrl(resolvedUrl);
+                return this.loader.resolve(request, undefined, resolvedUrl);
```
Comment (anthony-murphy) [inline packages/loader/container-loader/src/loader.ts:70]: don't use the loader here, just create the container

File: packages/loader/container-definitions/src/loader.ts [hunk 2]
```diff
@@ -196,7 +196,7 @@ export interface ILoader extends IFluidRouter {
      * An analogy for this is resolve is a DNS resolve of a Fluid container. Request then executes
      * a request against the server found from the resolve step.
      */
-    resolve(request: IRequest, pendingLocalState?: string): Promise<IContainer>;
+    resolve(request: IRequest, pendingLocalState?: string, resolvedUrl?: IResolvedUrl): Promise<IContainer>;
```
Comment (anthony-murphy) [inline packages/loader/container-definitions/src/loader.ts:199]: we shouldn't modify this, as its our public api for the loader

File: packages/loader/container-loader/src/loader.ts (hunk 1 above)
Comment (jatgarg) [reply packages/loader/container-loader/src/loader.ts]: Done.

File: packages/loader/container-definitions/src/loader.ts (hunk 2 above)
Comment (jatgarg) [reply packages/loader/container-definitions/src/loader.ts]: Removed.

**Review by anthony-murphy — APPROVED** [review]

**Pull request merged**